tests/core/test_configuration.py

```diff
@@ -1447,3 +1448,36 @@ def test_future_warning_only_for_code_ref(self, key):
             w = captured.pop()
             assert "your `conf.get*` call to use the new name" in str(w.message)
             assert w.category == FutureWarning
+
+
+def test_sensitive_values():
+    from airflow.settings import conf
+
+    # this list was hardcoded prior to 2.6.2
+    # included here to avoid regression in refactor
+    # inclusion of keys ending in "password" or "kwargs" is automated from 2.6.2
+    # items not matching this pattern must be added here manually
+    sensitive_values = {
+        ("database", "sql_alchemy_conn"),
+        ("core", "fernet_key"),
+        ("celery", "broker_url"),
+        ("celery", "flower_basic_auth"),
+        ("celery", "result_backend"),
+        ("atlas", "password"),
+        ("smtp", "smtp_password"),
+        ("webserver", "secret_key"),
+        ("secrets", "backend_kwargs"),
+        # The following options are deprecated
+        ("core", "sql_alchemy_conn"),
+        ("database", "sql_alchemy_engine_args"),
+        ("sentry", "sentry_dsn"),
```

Comment:
These aren't deprecated? Move them up.

There is probably a deprecated peer for the engine args though.

Reply:
fixed this; i don't think we need to keep track here which ones are deprecated or not.  it's just verifying that the full list is as expected.

sql_alchemy_engine_args was not renamed according to our structures that track that